cordformation/src/test/kotlin/net/corda/plugins/DockerformTest.kt

```diff
@@ -111,5 +114,23 @@ class DockerformTest : BaseformTest() {
         assertThat(getNodeCordappJar(bankOfCordaNodeName, cordaFinanceWorkflowsJarName)).isRegularFile()
         assertThat(getNodeCordappJar(bankOfCordaNodeName, cordaFinanceContractsJarName)).isRegularFile()
         assertThat(getNetworkParameterOverrides(bankOfCordaNodeName)).isRegularFile()
+
+        val notaryConfigPath = getNodeConfig(notaryNodeName)
+        assertThat(notaryConfigPath).isRegularFile()
+
+        val notaryConfig = ConfigFactory.parseFile(notaryConfigPath.toFile())
+        assertTrue(notaryConfig.hasPath("rpcSettings.address"))
+        assertTrue(notaryConfig.hasPath("rpcSettings.adminAddress"))
+        assertEquals(ConfigurationUtils.parsePort(notaryConfig.getString("rpcSettings.address")), 10003)
```

Comment:
Funny thing - this `assertEquals()` function is really:
```kotlin
assertEquals(expectedValue, actualValue)
```
i.e. the assertion should be written as:
```kotlin
assertEquals(10003, ConfigurationUtils.parsePort(notaryConfig.getString("rpcSettings.address")))
```
(FYI different frameworks put these parameters in different orders. I don't know why....)

Reply:
Fixed now :-)